src/content/samples/node.mdx

```diff
@@ -25,76 +25,65 @@ $ git clone https://github.com/okteto/node-getting-started
 $ cd node-getting-started
 ```
 
-The `k8s.yml` file contains the [Kubernetes manifests](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) of the Node.js Sample App.
-Deploy a dev version of the application by executing:
+At the root of the directory, you'll find the `okteto.yaml` file. This describes how to [build](/docs/reference/manifest/#build-object-optional) and [deploy](/docs/reference/manifest/#deploy-string-optional) the Node Sample App.
 
-```console
-$ kubectl apply -f k8s.yml
 ```
-
-```console
-deployment.apps "hello-world" created
-service "hello-world" created
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
```

Comment:
Let's don't show the `dev` section as we do here:
https://github.com/okteto/docs/pull/119/files#diff-a7c7d0def22235aa498659f147beb3ee4800dfde172e7f81d3ab29244224de16R38